src/pages/home/categories/categories.js

```diff
@@ -14,19 +14,16 @@ const Categories = () => {
   }));
   const styles = useStyles();
 
-  const CategoriesView = () => {
-    if (!categories) {
-      return null;
-    }
-    return categories.map(({ _id, name, images, categoryCode }) => (
+  const categoriesList = !categories
+    ? null
```

Comment:
Usually, it is better to have positive case first `categories ? categories.map(...) : null`

Reply:
Changed